### DISCUSSION TITLE
Skinning Idea: Accuracy number color tint

### DISCUSSION THREAD

**Discussion by Zylviij** [ppy/osu#16875] [Ideas]
Add a configuration option to the skin's config file that allows the accuracy text's color to be tinted based on the current accuracy. This would be nice for when you want to get a sense of the accuracy from peripheral vision, without having to explicitly look to read the value.

Maybe for the default skin, it could change the color to match the current letter grade.

**Comment by iSimpFor** [comment]
sounds like a good idea, maybe this can be applied to pp, score, and combo too

**Reply by dekrain** [reply]
and UR
